physionet-django/console/views.py

```diff
@@ -1004,6 +1017,15 @@ def user_management(request, username):
     """
     user = get_object_or_404(User, username__iexact=username)
 
+    _trainings = Training.objects.select_related('training_type').filter(user=user).order_by('-status')
```

Comment:
"trainings" isn't getting returned to the template as i think was intended. i have fixed this in a set of forthcoming updates (which mostly focus on the wording we use to describe credentialing and training).

Reply:
Yes, I also noticed that it always renders nothing.

I also think that ordering by status is not really necessary in this case since we display each status separately anyway:
```suggestion
    _trainings = Training.objects.select_related('training_type').filter(user=user)
```

Reply:
Anyway - thanks for handling that. I also see some very minor code style issues that might be improved (e.g. change the name from `trainings` to e.g. `trainings_by_status` so it's then clearer in the template, use a dict literal instead of assigning one-by-one) but this is very miniscule - if I don't make it then don't hesitate to merge IMHO.